db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/ServiceFilter.class/__methods__/filterbydialogparameters.rb

```diff
@@ -0,0 +1,72 @@
+# Description: This sample method allows the customer to control which services
+#	       get processed. By default it allows all services to be included
+#	       from the bundle. If the user wants to do be selective about which
+#	       services from a bundle get installed, they can copy this method and
+#	       apply custom logic
+#
+# This example is for bundled services, which is composed of multiple catalog items.
+# The root service is always included.
+
+# In this example we match the 'dialog_environment' to match the service_template name
+# for a service to be included.
+#
+
+# Input Parameters:
+#	   $evm.root['service_template']  contains the current Service Template Object being evaluated
+#	   $evm.root['service'] 	  contains the parent service object, will be nil if
+#					  this is the first service
+#      $evm.root['service_template_provision_task']
+#
+# Output : The evm.root should have an attribute called include_service which is
+#	   evaluated by the internal state machine to decide if the service gets
+#	   included or excluded
+#
+
+def dump_root
```

Comment:
I do not like littering the `dump_root` method all over the sample automate methods.  There is an example of it in the InspectMe method which users can add if needed.  Let's remove the method and the commented out call to it.